.aws/src/SqsLambda.ts

```diff
@@ -19,7 +19,7 @@ export class SqsLambda extends Resource {
     const { sentryDsn, gitSha } = this.getEnvVariableValues();
 
     new PocketSQSWithLambdaTarget(this, 'sqs-event-consumer', {
-      name: `${config.prefix}-Sqs-Event-Consumer`,
+      name: `${config.prefix}-sqs-event-consumer`,
```

Comment:
Pretty sure lowercasing here will screw up CI. Unless you update CI references, ex. https://github.com/Pocket/annotations-api/pull/207/files#diff-183e977459995ccde361fcaff10430e2bc23e5fb380c6e357c1abfc5cfb3c7d9R167

Reply:
I did update the CI reference, but I will double check the dev deployment